application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java

```diff
@@ -131,6 +143,80 @@ private RestAction<Message> sendExplanationMessage(GuildMessageChannel threadCha
         return action.setEmbeds(embeds);
     }
 
+    /**
+     * Determine between the title of the thread and the first message which to send to the AI. It
+     * uses a simple heuristic of length to determine if enough context exists in a question. If the
+     * title is used, it must also include a question mark since the title is often used more as an
+     * indicator of topic versus a question.
+     * 
+     * @param questionFirstMessage - The first message of the thread which originates from the
+     *        question asker.
+     * @param threadChannel - The thread in which the question was asked.
+     * @return An answer for the user from the AI service or a message indicating either an error or
+     *         why the message wasn't used.
+     */
+    RestAction<Message> constructChatGptAttempt(String questionFirstMessage,
+            ThreadChannel threadChannel) {
+        // Will be at most 100 characters.
```

Comment:
where is this 100 coming from? this comment will likely get out of date fast

Reply:
It comes from the allowable length of titles for threads on discord (atleast when I was generating a question via the discord). Is that number adjustable? I know discord could change it at anytime. I can remove the comment but I thought it was some nice context.

Reply:
im pretty sure that there would be a constant in JDA for this. anyways, my point is more that u have to write stuff like that or link it. otherwise people will be confused

Reply:
No that makes sense - If I need to make another change I will add it back.